elastic/integrations
# IIS ignore older option

- Enhancement

## What does this PR do?
Adds support to ignore logs older than specified period. Details are in https://github.com/elastic/integrations/issues/3147


## Checklist





## How to test this PR locally
Install IIS integration
Check for new `ignore_older `option is available for logs datastream.
Verify the log collection by specifying the value to the ignore_older option.




- Closes #3847
